src/app/admin/pages/learning-objects/learning-objects.component.html

```diff
@@ -26,8 +26,15 @@
             <div>Status</div>
             <div>Name</div>
             <div>Author</div>
-            <div>Type</div>
-            <div>Date</div>
+            <div>Length</div>
+            <div>Date
+              <button class="sort" (activate)="sortByDate(1)" aria-label="Sort learning objects in ascending order" [tip]="'Sort Ascending'" tipPosition="top" [tipDisabled]="false">
+                <i style="font-size: 18px" class="fas fa-angle-up"></i>
```

Comment:
Inline styles due to a deprecation in the font awesome package. We would need to update to do it from the stylesheet from what I've found online.